### PR TITLE
Add a round-robin baseline policy

### PR DESCRIPTION
**Pull request by Elarnon**
This adds a baseline policy using round-robin selection, i.e. which is
selecting the child which was visited least at each step.

### CODE REVIEW

File: src/explorer/mcts.rs [hunk 1]
```diff
@@ -2001,3 +2018,32 @@ impl<'a> IntoIterator for &'a Evaluations {
...
+        &'_ self,
+        _cut: f64,
+        view: &NodeView<'_, 'c, N, CommonStats>,
+    ) -> Option<(EdgeViewIndex, Selector<EdgeIndex>)> {
+        Selector::maximum(
+            view.iter()
+                .map(|(index, edge, _node)| (index, -(edge.data().num_visits() as f64)))
+                .collect(),
+        )
+        .map(|selector| {
```
Comment (nicoTolly) [inline src/explorer/mcts.rs]: I had a hard time understanding that this map is not the map implemented in Selector but the map of Option<Selector> (I finally guessed based on the closure argument being named selector, which induces the type). Maybe a comment clarifying what is happening would be worth here ? "Map Option<Selector<Node>> to Option<Selector<Edge>> "

Comment (Elarnon) [reply src/explorer/mcts.rs]: If this needs a comment everywhere that's not going to work.  I'd rather rename `Selector::maximum` to `Selector::try_maximum` (and `Selector::random` to `Selector::try_random` to be consistent) to show the result is not a `Selector` instance if that's OK with you.

Comment (nicoTolly) [reply src/explorer/mcts.rs]: Ok :)

**Comment by Elarnon** [comment]
bors r=nicoTolly

hmpf.

**Pull request merged**